examples/vnet/README.md

```diff
@@ -4,6 +4,14 @@
 
 These examples show you how to build a customized Docker enabled cluster on Microsoft Azure where you can provide your own VNET.
 
+By providing your own VNET, you can change the VNET address space from the default of 10.0.0.0/8 to a smaller subnet, like
+10.1.0.0/16. This may be useful if you have existing VNETs that you would like to peer with, or plan to have them in the future.
+Kubernetes will still use 10.0.0.0/8 internally within the cluster, but all the node ports will be within the modified VNET

```

Comment:
I would like to see acknowledgement of the 3 kubernetes networks .  Here's a proposal for the last sentence?

For Kubernetes, a change of the address space only affects the IP addresses assigned to the masters and the agents.  The service network of 10.0.0.0/16 is still implemented by the kube-proxy in each host, and the pod network stays as 10.244.0.0/16.  This means that access to services from another VNET will have to be done via node port through the host addresses.